src/dbFirebase.js

```diff
@@ -14,7 +14,7 @@ function extractPhoto(doc) {
   photo.id = doc.id;
   photo.updated = photo.updated && photo.updated.toDate();
   photo.moderated = photo.moderated && photo.moderated.toDate();
-
+  photo.moderator  = photo.moderator && photo.moderator.path;
```

Comment:
solving: Unhandled Rejection (TypeError): Converting circular structure to JSON

Reply:
it solves it but you lose the reference. Better to leave the reference here and add a map in https://github.com/Geovation/photos/blob/master/src/dbFirebase.js#L50